src/components/ItemTop.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import searchIcon from "../assets/ic_search@3x.png";
+import dropdownIconPC from "../assets/ic_dropdown@3x.png";
+import dropdownIconMobile from "../assets/ic_dropdown_mobile@3x.png";
+import "../styles/items.css";
+
+const H1 = ({ children }) => {
+  return <h1 className="h1__items">{children}</h1>;
+};
+
+const Search = () => {
+  return (
+    <form className="search">
+      <button className="search__button">
+        <img src={searchIcon} alt="search" width="15px" height="15px" />
+      </button>
+      <input
+        className="search__input"
+        placeholder="검색할 상품을 입력해주세요"
+      />
+    </form>
+  );
+};
+
+const LinkBtn = ({ children, to }) => {
+  return (
+    <Link to={to}>
+      <button className="regist">{children}</button>
+    </Link>
+  );
+};
+
+const DropDown = ({ filterName, filter, setFilter }) => {
+  const [dropdown, setDropdown] = useState(false);
+
+  return (
+    <div className="dropdown">
+      <button
+        className="filter__button button--pc"
+        onClick={() => {
+          setDropdown((v) => !v);
```

Comment:
```suggestion
          setDropdown((v) => !v);
```
어떤 데이터가 들어있을지를 변수 이름으로 알 수 있을텐데 v 라는 알파벳은 강우님만 알수 있는 표현같아요. 
함수형 업데이트 는 이전 상태를 인자로 받기때문에 일반적으로 prev, 혹은 prevState라고 써줍니다.

꼭 이 경우가 아니라도 콜백의 인자로 전달되는 매개변수의 이름은 다른 개발자가 봤을 때  어떤 데이터를 담고 있는지 
유추 하기 쉽게 이름을 `잘` 지어주는것도 좋은 방법중 하나입니다 :-)